packages/chart/src/components/Legend/helpers/index.ts

```diff
@@ -11,14 +11,19 @@ export const getGradientConfig = (config, formatLabels, colorScale) => {
 }
 
 export const getMarginTop = (isLegendBottom, config) => {
+  const DEFAULT_MARGIN_TOP = 27
```

Comment:
We need better documentation when random numbers are used. Can you add a comment on why 27 is used here? Is it derived from margins/padding?